packages/help-center/src/components/help-center-chat-history.tsx

```diff
@@ -55,6 +59,9 @@ const Conversations = ( {
 	return (
 		<>
 			{ conversations.map( ( conversation ) => {
+				const { numberOfUnreadMessages } = calculateUnread( [
+					conversation as ZendeskConversation,
```

Comment:
It's probably me not being familiar enough with TypeScript but what happens when the `conversation` is an `OdieConversation`?

Reply:
Well that's exactly my point - do we really need to cast that `conversation` into `ZendeskConversation`? If the conversation is of type `OdieConversation`, will it trigger a runtime error?

Reply:
Oh well, just checked it. The parameter is actually used in `calculateUnread`. `CalculateUnread` uses a property that `OdieConversation` is missing: `participants`. That's why It's casted here.

OdieConversation won't have unread messages as it's always replying immediately and `OdieConversation` not having the participants property, when casted to ZendeskConversation, returns 0.

I hope I was able to clarify it enough 😄